README.md

```diff
@@ -80,6 +80,24 @@ Simple example for creating two users and two groups.
               - ipsum
 ```
 
+Creating a jailed SFTP user:
+
+```YAML
+- name: Configure User Access
+  hosts: sandbox
+
+  roles:
+    - name: sansible.users_and_groups
+      users_and_groups:
+        authorized_keys_dir: /etc/ssh/authorized_keys
+        groups:
+          - name: sftp_only
+        users:
+          - name: sftp
+            group: sftp_only
+            home: /mnt/sftp_vol
+```
+
 In most cases you would keep the list of users in external vars file or
```

Comment:
I had a chat with @chroche and it all sounds feasible. However I would prefer a more detailed documentation.